app/controllers/resources_controller.rb

```diff
@@ -6,78 +6,20 @@ class ResourcesController < ApplicationController
   before_action :authorize_request
 
   # POST /objects
-  # rubocop:disable Metrics/AbcSize
-  # rubocop:disable Metrics/MethodLength
   def create
     authorize! :resource
-    begin
-      response_cocina_obj = Dor::Services::Client.objects.register(params: cocina_model)
-    rescue Dor::Services::Client::UnauthorizedResponse => e
-      return render build_error('Error registering object with dor-services-app', e, '500', true)
-    rescue Dor::Services::Client::UnexpectedResponse => e
-      return render build_error('Error registering object with dor-services-app', e, '502')
-    rescue Dor::Services::Client::ConnectionFailed => e
-      return render build_error('Unable to reach dor-services-app', e, '504', true)
-    end
-
-    # Doing this here rather than IngestJob so that have accurate timestamp for milestone.
-    begin
-      workflow_client.create_workflow_by_name(response_cocina_obj.externalIdentifier, 'registrationWF', version: 1)
-    rescue Dor::WorkflowException => e
-      return render build_error('Error creating registrationWF with workflow-service', e, '502')
-    end
-
-    result = BackgroundJobResult.create
-    IngestJob.perform_later(druid: response_cocina_obj.externalIdentifier,
-                            filesets: params[:structural].to_unsafe_h.fetch(:contains, []),
+    result = BackgroundJobResult.create(output: {})
+    IngestJob.perform_later(model_params: params.except(:action, :controller, :resource, :accession).to_unsafe_h,
```

Comment:
How about we pass the cocina model, that way the consumer at least gets fast validation of the structure of what they sent and that all the files they reference are something sdr-api is aware of.

Reply:
Won't this validation will already be provided by openapi / committee?

Reply:
I guess the structural validation would, but not necessarily "we know about these" files validation.

Reply:
What do you mean about the "we know these" files validation? That we can find an ActiveStorage blob for each Cocina file? If so, we could, but doesn't seem necessary to do early validation of a problem that we've never had and will catch anyway.